Bus Scheduler/src/components/tripSearch/store/types.ts

```diff
@@ -0,0 +1,23 @@
+
+export type TripListType =
+  {
+    data: {
```

Comment:
good to create a sub type and use it here

Reply:
Done